common/src/main/java/org/apache/celeborn/common/network/client/TransportClientFactory.java

```diff
@@ -84,9 +89,11 @@ private static class ClientPool {
   private EventLoopGroup workerGroup;
   protected ByteBufAllocator pooledAllocator;
 
-  public TransportClientFactory(TransportContext context) {
+  public TransportClientFactory(
+      TransportContext context, List<TransportClientBootstrap> clientBootstraps) {
     this.context = Preconditions.checkNotNull(context);
     TransportConf conf = context.getConf();
+    this.clientBootstraps = Lists.newArrayList(Preconditions.checkNotNull(clientBootstraps));
```

Comment:
```suggestion
    this.clientBootstraps = Arrays.asList(Preconditions.checkNotNull(clientBootstraps));
```

Reply:
@SteNicholas  `clientBootstraps` is a List. So, the above doesn't work. I'll change it back.

Reply:
```suggestion
    this.clientBootstraps = Preconditions.checkNotNull(clientBootstraps);
```
Could this update to above? IMO, this does not need to use `Lists.newArrayList`.